src/OrchardCore.Modules/OrchardCore.Contents/Services/DefaultContentsAdminListQueryService.cs

```diff
@@ -1,45 +1,105 @@
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
 using OrchardCore.ContentManagement;
 using OrchardCore.Contents.ViewModels;
 using OrchardCore.DisplayManagement.ModelBinding;
 using OrchardCore.Modules;
 using YesSql;
+using YesSql.Filters.Abstractions.Nodes;
 
 namespace OrchardCore.Contents.Services
 {
     public class DefaultContentsAdminListQueryService : IContentsAdminListQueryService
     {
+        public const string DefaultTermName = "text";
```

Comment:
Maybe define this constant in the options class instead (if we keep the options pattern).